Object Storage -  Patch s3 compliancy

lason-ovh opened the pull request:
<!--
Hello 👋
Thank you for submitting a Pull Request.

For Work In Progress Pull Requests, please use the Draft PR feature, see https://github.blog/2019-02-14-introducing-draft-pull-requests/ for further details. As long as your Pull Request is a draft, the OVHcloud Guides Team will not start proofreading it.

For a timely review/response, please avoid force-pushing additional commits if your PR already received reviews or comments.

DO NOT USE a fork if you are an OVHcloud employee. Instead, please contact the OVHcloud Guides Team so to be granted /write access to this repository.

Feel free to apply labels on your Pull Request, selecting them from the list to your right.

Before submitting a Pull Request, please ensure you have:

- 📖 Read the OVHcloud Documentation readme: https://github.com/ovh/docs/blob/develop/readme.md
- 🇬🇧 Edited (at least) the en-gb version of a guide.
- 🇫🇷 Edited also the fr-fr version if you are a French speaker.
- 🌐 Edited all the files if you're doing a fix or a minor update (this is not mandatory but recommended).
- 👷‍♀️ Created a small PR, if applicable.
- ✅ Tested every step you're documenting.
- 📝 Used descriptive commit messages.
- 📐 Resized images which width exceeds 1400px.
- 📝 Edited or blurred any sensitive/private information from your text and images (IPs, user IDs, private URLs, etc.).
-->

## What type of Pull Request is this?

<!-- Delete the lines which don't apply: -->
- Fix

## Description
There were missing http verbs that we normally support in pre-signed urls. Also, we added a new sub-category for POST because it is a feature on its own and to eliminate confusion.

## Mandatory information

<!-- Delete the line which doesn't apply: -->
- This Pull Request can be merged as soon as possible AFTER reviews from @agoude 

<!-- If you know about the following, please mention it. If you don't know, delete the line.-->
- This Pull Request content should be replicated for the US OVHcloud documentation : YES

lason-ovh committed:
Update guide.fr-fr.md

ajout des verbes http manquants

lason-ovh committed:
Update guide.en-gb.md

added missing supported http verbs

DoStupidThingsFasterWithMoreEnergy commented:
To be consistent with the issue mentioned here: https://github.com/ovh/docs/issues/7874, perhaps replace the term `Pre-signed urls` with `Presigned urls`, which is the term most commonly used by AWS. This would allow this guide to be displayed in the search results for the `presign` function, which is not mentioned in this guide.

![pre-signed_query](https://github.com/user-attachments/assets/0bbbd380-c882-4fa2-bee7-b4f1b46dbc59)

lason-ovh committed:
Update guide.en-gb.md

presigned instead of pre-signed

lason-ovh committed:
Update guide.fr-fr.md

presigned instead of pre-signed

lason-ovh commented:
I replaced pre-signed with presigned according to preconisations

lason-ovh committed:
Add files via upload

ajout des screenshots FR

lason-ovh committed:
Update guide.en-gb.md

changed title indentation for Offsite Replication

lason-ovh committed:
Update guide.fr-fr.md

modification de l'indentation du titre pour Offsite repli

lason-ovh committed:
Delete pages/storage_and_backup/object_storage/s3_asynchronous_replication/images/replication-rule-creation-FR.PNG

deleted rule creation FR

lason-ovh committed:
Delete pages/storage_and_backup/object_storage/s3_asynchronous_replication/images/replication-rules-delete-FR.png

screenshot has not supported edit functionality

lason-ovh committed:
Add files via upload

lason-ovh committed:
Update guide.fr-fr.md

ajout de la partie UI pour configurer une règle de répli
/!\ il manque encore un screenshot

lason-ovh committed:
Update guide.en-gb.md

added instructions to create a replication rule using the UI
/!\ screenshots in EN have yet to be added

lason-ovh committed:
Add files via upload

added rule creation screenshot in FR

lason-ovh committed:
Update guide.fr-fr.md

ajout du screenshot manquant

lason-ovh committed:
Add files via upload

added EN screeshots

lason-ovh committed:
Update guide.en-gb.md

added missing screenshots

agoude commented on pages/storage_and_backup/object_storage/s3_asynchronous_replication/guide.en-gb.md:
````diff
@@ -440,7 +440,61 @@ $ aws s3api put-bucket-replication --bucket <source> --replication-configuration
 ```
 
-### Offsite Replication option in 3-AZ regions
+### Using the web UI
````
prefer using the term "OVHcloud Control Panel"

lason-ovh committed:
Update guide.fr-fr.md

utiisation de panneau de ctrl au lieu de web ui

lason-ovh committed:
Update guide.en-gb.md

using Ctrl panel instead of web ui

Y0Coss committed:
Proofreading - Duplication - Reducing images size to 1400px

Y0Coss committed:
S3 compliancy duplication & date update + Fixing colours on images

Y0Coss commented:
Thank you for your contribution @lason-ovh
